plugins/modules/clc_aa_policy.py

```diff
@@ -38,28 +38,24 @@
     type: str
     required: false
     default: present
-    choices: ['present','absent']
+    choices: ['present', 'absent']
 requirements:
-    - python = 2.7
-    - requests >= 2.5.0
-    - clc-sdk
+  - python = 2.7
```

Comment:
If the clc modules are really Python 2.7 only, we need to deprecate and remove them eventually. It seems that clc-sdk only supports Python 3 for the v2 API (https://github.com/CenturyLinkCloud/clc-python-sdk/blob/master/src/setup.py#L38); but from the notes below it seems that it supports the V2 API, since the environment variables and URLs include `v2`. So let's remove this.
```suggestion
```